src/components/DatePickerModal/readme.md

```diff
@@ -1,72 +1,312 @@
 ##### DatePickerModal base:
 
 ```js
-import { Button, DatePickerModal } from 'react-rainbow-components';
+import React from 'react';
+import { Picklist, Option, DatePickerModal } from 'react-rainbow-components';
 
-initialState = { isOpen: false };
+initialState = { isOpen: false, selection: { name: '', label: '' } };
 
-function getDate(value, selectionType) {
+function getDates(value, selectionType) {
     if (selectionType === 'single' && Array.isArray(value)) return value[0];
     return value;
 }
 
-<div
-    className="rainbow-align-content_center rainbow-m-vertical_large rainbow-p-horizontal_small rainbow-m_auto rainbow-flex"
->
-    <div className="rainbow-p-horizontal_medium">
-        <Button
-            label="Click to select a date"
-            onClick={() => setState({ isOpen: true, selectionType: 'single', value: null, title: 'Select a date' })} />
-    </div>
-    <div className="rainbow-p-horizontal_medium">
-        <Button
-            label="Click to select a range of dates"
-            onClick={() => setState({ isOpen: true, selectionType: 'range', value: null, title: 'Select range of dates' })} />
-    </div>
+function setDates(value) {
+    if (value) {
```

Comment:
sometimes we need to do this kind of checks but in this case, the Picklist always will send a value in the onChange then there is no need to do this check here